all-actors/src/main/java/org/sunbird/utilities/CertificateUtil.java

```diff
@@ -52,6 +53,15 @@ public static boolean isIdPresent(String certificateId) {
         }
         return false;
     }
+    public static Response deleteRecord(String id) throws BaseException {
+        Response response = cassandraOperation.deleteRecord(JsonKeys.SUNBIRD,JsonKeys.CERT_REGISTRY,id);
```

Comment:
I think in certificate deleted we need to make sure oldId and userId combination need to be found.
Otherwsie delete can have problem.

Reply:
But recepient identifier is not mandatory.

Reply:
done